### PR TITLE
Separate languages and GIFS

### PR DESCRIPTION
**Pull request by joextodd**
- Add some GIFS
- Add Messenger app
- Add Kotlin demo app
- Use chirpsdk from gradle

Closes: #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +56,6 @@ google-services.json
 freeline.py
 freeline/
 freeline_project_description.json
```
Comment (dinuchiriac) [inline .gitignore]: What is `freeline` and do we need to have this in `.gitignore`?

File: Java/Demo/app/build.gradle [hunk 2]
```diff
@@ -19,16 +19,14 @@ android {
 }
 
 repositories {
-    flatDir {
-        dirs 'libs'
+    maven {
+        url "https://maven.chirp.io/release"
```
Comment (dinuchiriac) [inline Java/Demo/app/build.gradle]: Can we have this in the project Gradle file instead?

File: Kotlin/Demo/app/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (dinuchiriac) [inline Kotlin/Demo/app/.gitignore]: This can probably go in the root `.gitignore`, what do you think?

File: .gitignore (hunk 1 above)
Comment (joextodd) [reply .gitignore]: No idea, just copied the current android demos `.gitignore`

File: Kotlin/Messenger/app/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (dinuchiriac) [inline Kotlin/Messenger/app/.gitignore]: same here,  maybe we can add this in root `.gitignore` instead?

File: .gitignore (hunk 1 above)
Comment (dinuchiriac) [reply .gitignore]: `Freeline is a super fast build tool for Android and an alternative to Instant Run` - never heard about this and I don't think we are using it anywhere here.

**Pull request merged**